src/parser.rs

```diff
@@ -124,26 +124,26 @@ where
     }
 
     /// Register non-standard filters
-    #[cfg(not(feature = "extra-filters"))]
+    #[cfg(not(feature = "all"))]
     pub fn extra_filters(self) -> Self {
         self
     }
 
     /// Register non-standard filters
-    #[cfg(feature = "extra-filters")]
+    #[cfg(feature = "all")]
     pub fn extra_filters(self) -> Self {
         self.filter(filters::extra::DateInTz)
             .filter(filters::extra::Pluralize)
     }
 
     /// Register non-standard filters
-    #[cfg(not(feature = "jekyll-filters"))]
+    #[cfg(not(feature = "jekyll"))]
     pub fn jekyll_filters(self) -> Self {
         self
     }
 
     /// Register non-standard filters
-    #[cfg(feature = "jekyll-filters")]
+    #[cfg(feature = "jekyll")]
     pub fn jekyll_filters(self) -> Self {
         self.filter(filters::jekyll::Slugify)
             .filter(filters::jekyll::Pop)
```

Comment:
There are no all or jekyll features.  Instead these functions should be deleted.